datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -248,9 +245,12 @@ def testable(ctx, start_id, agent_version, milestone, dry_run):
         pr_author = pr_data.get('user', {}).get('login', '')
         pr_body = pr_data.get('body', '')
 
-        teams = [trello.label_team_map[label] for label in pr_labels if label in trello.label_team_map]
+        if not (user_config['jira']['user'] and user_config['jira']['user']):
```

Comment:
Let's save `user_config['jira']` to a variable

Reply:
Whoops, this was actually a typo- should be  `if not (user_config['jira']['user'] and user_config['jira']['token']):`

Reply:
Fixed!